syz-manager/rpc.go

```diff
@@ -83,11 +85,44 @@ func startRPCServer(mgr *Manager) (*RPCServer, error) {
 	return serv, nil
 }
 
+var (
+	once                sync.Once
+	coverFilterFilename string
+	enabledCoverFilter  bool
+)
+
+func (serv *RPCServer) update(a *rpctype.ConnectArgs, r *rpctype.ConnectRes) error {
+	var (
+		err         error
+		coverFilter map[uint32]uint32
+	)
+	once.Do(func() {
+		err = serv.mgr.updateKernelModules(a)
+		if err != nil {
+			return
+		}
+		coverFilterFilename, coverFilter, enabledCoverFilter, err = serv.mgr.updateCoverFilter()
+	})
+	if err != nil {
+		return err
+	}
+	if coverFilterFilename != "" {
+		data, err := ioutil.ReadFile(coverFilterFilename)
```

Comment:
It's not reasonable to save the bitmap to file just read it back few functions later. Make createCoverageFilter return []byte directly, it will make things simpler.